src/core-layers/polygon-layer/polygon-layer.js

```diff
@@ -160,8 +160,8 @@ export default class PolygonLayer extends CompositeLayer {
         miterLimit: lineMiterLimit,
 
         getPath: x => x.path,
-        getColor: getLineColor,
-        getWidth: getLineWidth,
+        getColor: x => getLineColor(x.object),
```

Comment:
@georgios-uber As far as I can tell this would make the layer look for `<object>.object.fillColor` which is inconsistent with the docs that says we are looking for `<object>.fillColor` and would be a big change. 

https://uber.github.io/deck.gl/#/documentation/layer-catalog/polygon-layer?section=render-options

Maybe something in the docs is unclear?

Reply:
This layer takes paths as input, not the original user-provided objects.
`const {paths} = this.state;`

This is generated in line 74 of this file.

The current implementation violates the docs (because fillColor is ignored) and this fixes it.